src/components/ShowReviews/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from "react";
+import style from "./style.module.scss";
+import Image from "next/image";
+import SendReview from "../SendReview";
+import { ReviewProps } from "@/pages/movies/[id]";
+
+export default function ShowReviews({ reviewData }: ReviewProps) {
+  const [isOpen, setIsOpen] = useState(true);
+
+  const toggleDropdown = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <>
+      {<button onClick={toggleDropdown}>testknapp</button>}
+      {!isOpen && (
+        <>
```

Comment:
This react fragment should be unnecessary. I always encourage removing unnecessary fragments as they make the code unnecessarily "wide".